geomesa-accumulo/geomesa-accumulo-jobs/src/main/scala/org/locationtech/geomesa/jobs/accumulo/AccumuloJobUtils.scala

```diff
@@ -122,7 +122,7 @@ object AccumuloJobUtils extends LazyLogging {
 
       val queryPlans = ds.getQueryPlan(query)
       if (queryPlans.isEmpty) {
-        Seq(EmptyPlan(FilterStrategy(fallbackIndex, None, Some(Filter.EXCLUDE), temporal = false, 0L)))
+        Seq(EmptyPlan(FilterStrategy(fallbackIndex, None, Some(Filter.EXCLUDE), temporal = false, Float.PositiveInfinity)))
```

Comment:
What is positive infinity representing?  

I think I'm obligated to ask whenever a number changes by an infinity.

Reply:
it's basically equivalent to Long.MaxValue, i.e. the lowest possible priority. The math works out if you use infinity :)